### PR TITLE
 English translation Clean Up

### PR DESCRIPTION
**Pull request by toehead2001**
The question marks always seemed silly to me. "Graphics" instead of "GFX", as it is more widely understood.

### CODE REVIEW

**Comment by MELERIX** [comment]
for consistency you also have to edit the strings in this file: https://github.com/project64/project64/blob/master/Source/Project64/Multilanguage/Language%20Class.cpp

File: Lang/English.pj.Lang [hunk 1]
```diff
@@ -255,8 +255,8 @@
 #503#  "Self Mod Methods"
 #504#  "Default Memory Size:"
 #505#  "Advanced Block Linking:"
-#506#  "Start Emulation when rom is opened?"
-#507#  "Always overwrite default settings with ones from ini?"
+#506#  "Start Emulation when rom is opened"
+#507#  "Always override default settings with ones from ini"
```
Comment (MELERIX) [inline Lang/English.pj.Lang]: maybe the right word here should be "replace" ?

**Comment by toehead2001** [comment]
I think "Overwrite" and "Replace" will make the user think that the settings in the rom ini will become the emulator defaults...  which would be silly
"Override" would make them understand that the emulator defaults will be bypassed and the settings in the ini will be used instead.

You make a valid point about the hyphen. Perhaps it would be simpler to just to use the abbreviated forms (LLE & HLE) everywhere.

Thanks for letting me know about that other file.

**Comment by toehead2001** [comment]
I made some revisions last night. Could I get this reviewed again?

**Comment by toehead2001** [comment]
I just noticed that "ROM" has inconsistent casing. We have "rom", "Rom", and "ROM".

I would think "ROM" and "ROMs" (for plural) would be the correct casing. Correct?

**Comment by cxd4** [comment]
Yes.  "ROM" is an acronym, and with rare exceptions, every letter in the acronym must be uppercase.

Even "rom" looks better than "Rom" ... really no reason to only uppercase the "read" part of the acronym.

**Comment by oddMLan** [comment]
>FireFox

Not meant to derail the main discussion, but it's actually spelled _Fire**f**ox_ :P

**Comment by toehead2001** [comment]
Another question.
We currently have "Instant Save", "Save State", and "State Save"

Do we want to go with "Save State" ? and if so, "Load State" or "Load Save State" ?

Personal, I prefer "Load State" and "Save State".

**Comment by MELERIX** [comment]
in others emulators such as: Snes9x & VBA-M it says "Load Game" & "Save Game".

but, technically we are not saving & loading using the game, we are saving & loading using a CPU state, which is practically a "state" of the CPU instructions in a specific moment.

so technically "Save State" & "Load State" should be the right words, I guess.

it require a debate xD

**Comment by cxd4** [comment]
Yeah, prefer as-is.  I usually just call them "save**d** states", not "save states".  As in the state of the machine...I'd hardly think of "save states" as a noun.  So actually I'd prefer the current way pj64 names that.

**Comment by toehead2001** [comment]
>So actually I'd prefer the current way pj64 names that.

PJ64 currently has three ways of naming them... hence my question.

**Pull request merged**